addons/web/static/src/webclient/settings_form_view/settings/setting_header.xml

```diff
@@ -5,7 +5,7 @@
             <div class="col-xs-12 col-md-6 ms-0 o_setting_box">
                 <div class="o_setting_right_pane border-start-0 ms-0 ps-0">
                     <div class="content-group">
-                        <div class="row flex-row flex-nowrap mt8 align-items-center">
+                        <div class="row flex-md-nowrap mt8 align-items-center">
```

Comment:
remove flex-md-nowrap as this is already the default value?

Reply:
Should be fixed in 16

Reply:
> remove flex-md-nowrap as this is already the default value?

sorry I didn't see the row class.

Reply:
> Should be fixed in 16

Thanks for review :)
I thought we can't remove class in stable because it might break customisation. I don't it'll work without removing  `flex-nowrap` that's why I didn't fix it in 16.0